plugins/quickinspector/quickoverlay.cpp

```diff
@@ -45,6 +45,49 @@
 #include <cmath>
 
 namespace GammaRay {
+
+class QQuickItemPropertyCache {
+public:
+    static const QQuickItemPropertyCache &getPropertyCache(QQuickItem *item)
+    {
+        static QHash<quintptr, QQuickItemPropertyCache> s_cache;
```

Comment:
Can't we actually use QMetaObject* here now, instead of quintptr? That would avoid all the ugly reinterpret_casts below, and make it much clearer what the key actually is.